bundler.gemspec

```diff
@@ -32,4 +32,6 @@ Gem::Specification.new do |s|
   s.bindir        = "exe"
   s.executables   = %w(bundle bundler)
   s.require_paths = ["lib"]
+
+  s.post_install_message = "Bundler and RubyGems.org are free for anyone to use, but maintaining them costs more than $20,000 USD every month. If you make a profit using Ruby, help us cover those costs so that we can keep the gem ecosystem free for everyone: https://rubytogether.org/join"
```

Comment:
@hmistry I also prefer the more subtle type of message, but my experience when talking to Ruby devs is that they say things like "wow I had no idea it was so expensive, or that RubyGems is growing so much it is in danger unless it gets more funding". If saying "please help us" doesn't include explicit mentions of the high cost or that this funding is the only thing making it possible at all, it seems like most if not all devs will just assume everything is fine and ignore it. :/